metagraph/src/graph/alignment/aligner_extender_methods.cpp

```diff
@@ -660,8 +660,8 @@ ::extend_main(std::function<void(DBGAlignment&&, NodeType)> callback,
         update_columns(node, out_columns, min_path_score);
     }
 
-    logger->trace("Extension completed:\tquery size:\t{}\tseed size:\t{}\texplored nodes:\t{}",
-                  query.size(), path_->size(), dp_table.size());
+    // logger->trace("Extension completed:\tquery size:\t{}\tseed size:\t{}\texplored nodes:\t{}",
+                  // query.size(), path_->size(), dp_table.size());
```

Comment:
I'll add a comment here. These produce way too much output in verbose mode, so I disabled them until we support variable verbosity levels.

Reply:
info / trace is not enough?

Reply:
one thing I noticed is that there's an extra level between info and trace that we're not using. What if we were to use debug in most cases and only use trace in places where it would produce a ton of output.

Then, we can make `-v` output debug and `-vv` output trace

Reply:
I don't use logger for debugging, so I never thought about this. Do you really need that?

Reply:
If an alignment takes too long, I like to use this debugging information to help me figure out why.

Reply:
Actually, I think for now I'll just wrap these in `#ifndef NDEBUG` as a workaround